hw2/04-background.html

```diff
@@ -9,8 +9,8 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="04-background.js"></script>
   </head>
-  <body>
-    <main>
+  <body class="container mt-5">
+    <main class=" bg-light rounded p-4 d-flex flex-column max-sm-75">
```

Comment:
Sizing of the container is a little strange here, as it doesn't match the example, try removing the classes from body and replace the classes in main with `card d-flex w-50 mx-auto mt-5 p-3`.